app/crash-herald.js

```diff
@@ -2,10 +2,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-const buildConfig = require('../js/constants/buildConfig')
 const Channel = require('./channel')
+const {app} = require('electron')
 
-const version = buildConfig.BROWSER_LAPTOP_REV || 'unknown'
```

Comment:
BROWSER_LAPTOP_REV is the ref of the last git commit

Reply:
+1 on what @aekeus said- was this intended?  @bbondy when testing the PR, does a commit hash still show on `about:brave`?  

I'm also curious why we'd default channel to empty string, rather than dev?

Reply:
just noticed this is `crash-herald.js`.  Maybe in this context (for released builds), the revision doesn't matter (especially since we now version via Jenkins).